nim/httpbeast/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nimlang/nim:0.20.2
+
+WORKDIR /usr/src/app
+
+COPY server.nim server.nimble ./
+
+RUN nimble c -d:release --threads:on -y server.nim
```

Comment:
for extra speed nowadays, you can do ``-d:danger`` ;)

Reply:
if you are OK with that I'll mark this case as resolved without doing anythin
in the next `PR` I add the options @see https://github.com/the-benchmarker/web-frameworks/pull/1578/files#diff-80d3cc452e003fc3abf65e23df5f9256
the purpose is to compile `jester` and `httpbeast` the same way

Reply:
do it whichever way you prefer, just note that `-d:danger` will give the best performance on latest Nim.